scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -8,6 +8,8 @@ workflow GvsExtractCallset {
     String dataset_name
     String project_id
 
+    String cohort_project_id = project_id
```

Comment:
PrepRanges initializes the cohort tables into `destination_*`; then this workflow assumed the cohort tables are in the GVS dataset.